tests/Ilios/MeSH/Model/BaseTestCase.php

```diff
@@ -0,0 +1,242 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Ilios\MeSH\Model;
+
+use DateTime;
+use Exception;
+use PHPUnit\Framework\TestCase;
+use Mockery as m;
+use Faker\Factory as faker;
+
+/**
+ * Class BaseTest
+ *
+ * @package Ilios\MeSH\Model
+ */
+abstract class BaseTestCase extends TestCase
+{
+    public function tearDown(): void
+    {
+        m::close();
+    }
+
+    protected function getArrayOfMockObjects(string $className, int $count): array
+    {
+        $arr = [];
+        for ($i = 0; $i < $count; $i++) {
+            $arr[] = m::mock($className);
+        }
+
+        return $arr;
+    }
+
+    /**
+     * @param  string $property
```

Comment:
docblocks in this file are duplicates of declarations in the code. Just remove them?

Reply:
done. :heavy_check_mark: